models/orogen/joint_dispatcher.rb

```diff
@@ -1,4 +1,5 @@
 require 'models/blueprints/control'
+using_task_library 'joint_dispatcher'
```

Comment:
This is wrong. It means someone is requireing models/orogen/joint_dispatcher manually which should never be done.